tools/ARIAtools/tsSetup.py

```diff
@@ -301,6 +307,8 @@ def main(inps=None):
         if layers!=[]:
             print('\nExtracting optional, user-specified layers %s for each interferogram pair'%(layers))
             export_products(standardproduct_info.products[1], standardproduct_info.bbox_file, prods_TOTbbox, layers, dem=demfile, lat=Latitude, lon=Longitude, mask=inps.mask, outDir=inps.workdir, outputFormat=inps.outputFormat, stitchMethodType='overlap', verbose=inps.verbose, num_threads=inps.num_threads, multilooking=inps.multilooking)
+    else:
+        inps.layers = []
```

Comment:
isn't this else obsolete?

Reply:
@ehavazli no, it's needed for the last check, which generates an amplitude stack only if the user specifies the download of amplitude layers (either explicitly, or by specifying 'all' as an argument). Makes sense?